onnx_tf/handlers/backend/conv_integer.py

```diff
@@ -0,0 +1,60 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from .conv_mixin import ConvMixin
+
+
+@onnx_op("ConvInteger")
+class Conv(ConvMixin, BackendHandler):
+
+  @classmethod
+  def _apply_x_zero_point(cls, base, zero_point):
+    base = tf.cast(base, tf.float32)
+    zero_point = tf.cast(zero_point, tf.float32)
+    return (base - zero_point)
+
+  @classmethod
+  def _apply_w_zero_point(cls, base, zero_point):
+    if len(zero_point.shape) == 0:
+      w_zero_point = cls._apply_x_zero_point(base, zero_point)
+    elif len(zero_point.shape) == 1:
+      for i in range(base.shape.as_list()[0]):
+        for j in range(zero_point.shape.as_list()[0]):
+          tensor = cls._apply_x_zero_point(base[i][j], zero_point[j])
```

Comment:
Why do you need the j loop here? Base on the spec, I believe each element in the 1-D tensor w_zero_point should apply to the corresponding M in w